DESCRIPTION

```diff
@@ -1,11 +1,12 @@
 Package: drat
 Type: Package
 Title: 'Drat' R Archive Template
-Version: 0.1.6
-Date: 2020-05-29
+Version: 0.1.6.4
+Date: 2020-06-09
```

Comment:
I told @pat-s for "mucking" with Version and Date as that is what the release manager aka maintainer does.  But at least you used the pattern I use which is a minimal delta increment (whereas he forcefully suggested that his changes should bump immediately to 0.2.0 which is .... something we agreed to reverse).   This is fine.